lms/djangoapps/instructor_task/tasks_helper/module_state.py

```diff
@@ -221,6 +225,75 @@ def rescore_problem_module_state(xmodule_instance_args, module_descriptor, stude
 
 
 @outer_atomic
+def override_problem_score_module_state(xmodule_instance_args, module_descriptor, student_module, task_input):
+    '''
+    '''
+    # unpack the StudentModule:
+    course_id = student_module.course_id
+    student = student_module.student
+    usage_key = student_module.module_state_key
+
+    with modulestore().bulk_operations(course_id):
+        course = get_course_by_id(course_id)
+        # TODO: Here is a call site where we could pass in a loaded course.  I
```

Comment:
This comment was a leftover from copying the rescore task above it. Looking into it, the comment is over two years old so I'm just going to delete it. I'm not sure it applies anymore since grading now happens in a totally different manner from two years ago.